src/render/src/device.rs

```diff
@@ -515,6 +514,28 @@ impl<B: Backend> Device<B> {
     {
         init.create(self, shader_entries, primitive, rasterizer)
     }
+
+    // TODO?: typed
+    pub fn create_framebuffer<P>(
+        &mut self,
+        pipeline: &P,
+        rtvs: &[&handle::raw::RenderTargetView<B>],
+        dsvs: &[&handle::raw::DepthStencilView<B>],
+        extent: Extent,
+    ) -> handle::raw::FrameBuffer<B>
+        where P: pso::GraphicsPipelineMeta<B>
+    {
+        let rtv_res: Vec<_> = rtvs.iter().map(|&rtv| rtv.resource()).collect();
+        let dsv_res: Vec<_> = dsvs.iter().map(|&dsv| dsv.resource()).collect();
+        let buffer = self.raw.create_framebuffer(
+            pipeline.render_pass(), &rtv_res[..], &dsv_res[..], extent);
+        let info = handle::FrameBufferInfo {
+            rtvs: rtvs.iter().map(|&rtv| rtv.clone()).collect(),
```

Comment:
could just have used `rtvs.iter().cloned().collect()` ?

Reply:
That would require `cloned().cloned()` because this is a `&&RTV`.